cidr/cidr_test.go

```diff
@@ -95,6 +96,91 @@ func TestSubnet(t *testing.T) {
 	}
 }
 
+func TestSubnetBig(t *testing.T) {
+	cases := []struct {
+		Base   string
+		Bits   int
+		Num    *big.Int
+		Output string
+		Error  bool
+	}{
+		{
+			Base:   "192.168.2.0/20",
+			Bits:   4,
+			Num:    big.NewInt(int64(6)),
+			Output: "192.168.6.0/24",
+		},
+		{
+			Base:   "192.168.2.0/20",
+			Bits:   4,
+			Num:    big.NewInt(int64(0)),
+			Output: "192.168.0.0/24",
+		},
+		{
+			Base:   "192.168.0.0/31",
+			Bits:   1,
+			Num:    big.NewInt(int64(1)),
+			Output: "192.168.0.1/32",
+		},
+		{
+			Base:   "192.168.0.0/21",
+			Bits:   4,
+			Num:    big.NewInt(int64(7)),
+			Output: "192.168.3.128/25",
+		},
+		{
+			Base:   "fe80::/48",
+			Bits:   16,
+			Num:    big.NewInt(int64(6)),
+			Output: "fe80:0:0:6::/64",
+		},
+		{
+			Base:   "fe80::/49",
+			Bits:   16,
```

Comment:
With `SubnetBig` now supporting Bits > 32, perhaps we could show that here with a non-failing extension of 33 bits, which I think would then produce an `/82` prefix in the result.